sdcm/scylla_bench_thread.py

```diff
@@ -251,7 +251,7 @@ def _run_stress(self, loader, loader_idx, cpu_idx):  # pylint: disable=too-many-
             except Exception as exc:  # pylint: disable=broad-except  # noqa: BLE001
                 self.configure_event_on_failure(stress_event=scylla_bench_event, exc=exc)
 
-        return loader, result
+        return loader, result, scylla_bench_event
```

Comment:
Maybe we need to align this on all stress commands ?

Reply:
This new `stop_nemesis_on_stress_errors` method is now applied to results of stress commands that are triggered from nemesis.py. There are 3 commands that are executed this way in nemesis code - c-s, s-b, and cql-stress-cassandra-stress.
All of them (with this^ change included) are returning tuple of (loader, result, event).

I'm not sure, if/which stress commands are also triggered from nemesis.py.

Reply:
I think regardless of what is being used right now, it would be good to align them  to return identically from this function.

Reply:
Agree, would be less confusion if we had single return type for all cases.